REST_API_DOCS.md

```
@@ -0,0 +1,1861 @@
+}
+```
+
+---
+
+### 좋아요 취소
+
+**POST** `/posts/{post_id}/likes`
+
+(**같은 엔드포인트 재사용**)로그인한 사용자가 특정 게시글에 좋아요를 취소합니다. 이미 좋아요를 취소한 게시글에 다시 요청할 경우 좋아요가 등록됩니다.
+
+**Request Headers**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)